zenaton/traits/with_duration.py

```diff
@@ -38,7 +38,7 @@ def weeks(self, value):
     # Inspired by https://stackoverflow.com/a/50301887
     def months_to_days(self, months):
 
-        now = datetime.datetime.today().date()
```

Comment:
Evaluating `self.years(1)` twice would actually add the upcoming year twice instead of adding the upcoming one, then the one after. So here, when evaluating how long is a month we start at the `then` time instead of `now`.